### ISSUE TITLE
No provider for MdDataTable!

### ISSUE THREAD

**Issue by szechyjs** [justindujardin/ng2-material#133]
After upgrading to 0.3.3 I now get the following error on load.

```
angular2.dev.js:23730 EXCEPTION: Error: Uncaught (in promise): No provider for MdDataTable! (MdDataTableTr -> MdDataTable)
```

package.json
```json
"dependencies": {
    "angular2": "2.0.0-beta.12",
    "systemjs": "0.19.24",
    "es6-promise": "^3.0.2",
    "es6-shim": "^0.35.0",
    "reflect-metadata": "0.1.2",
    "rxjs": "5.0.0-beta.2",
    "zone.js": "0.6.6",
    "ng2-material": "^0.3.3"
  },
  "devDependencies": {
    "concurrently": "^2.0.0",
    "jasmine-core": "2.4.1",
    "lite-server": "^2.1.0",
    "typescript": "^1.8.9",
    "typings": "^0.7.9"
  }
```

**Comment by justindujardin** [comment]
@szechyjs Please provide a plunkr reproducing the issue.

/fyi @Gregcop1

**Comment by szechyjs** [comment]
After much playing around I've finally found what reproduces the issue.

Simply add a table and a row to a component.

http://plnkr.co/edit/KjZ2c4CS6UKI7NFuOOxW?p=preview

**Comment by justindujardin** [comment]
Fixed with 0.3.4, and deployed to npm.
